functional/FunctionalTest.java

```diff
@@ -3798,35 +3795,20 @@ public static void getBucketPolicy_test1() throws Exception {
               + "\"Principal\":{\"AWS\":[\"*\"]},\"Resource\":[\"arn:aws:s3:::"
               + bucketName
               + "/myobject*\"],\"Sid\":\"\"}]}";
-      client.setBucketPolicy(bucketName, policy);
-      client.getBucketPolicy(bucketName);
-      mintSuccessLog("getBucketPolicy(String bucketName)", null, startTime);
+      client.setBucketPolicy(
+          SetBucketPolicyArgs.builder().bucket(bucketName).config(policy).build());
+      client.getBucketPolicy(GetBucketPolicyArgs.builder().bucket(bucketName).build());
```

Comment:
Maybe we should verify that the value returned by `getBucketPolicy` matches what was set in the previous step?

Reply:
I think it is hard to check. For example below two JSON string are same.
```json
{"Version":"2012-10-17","Statement":[{"Action":["s3:GetObject"],"Effect":"Allow","Principal":{"AWS":["*"]},"Resource":["arn:aws:s3:::MYBUCKET/myobject*"],"Sid":""}]}
```

```json
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Action": [
                "s3:GetObject"
            ],
            "Effect": "Allow",
            "Principal": {
                "AWS": [
                    "*"
                ]
            },
            "Resource": [
                "arn:aws:s3:::MYBUCKET/myobject*"
            ],
            "Sid": ""
        }
    ]
}
```
Server may return a JSON which can be formatted differently but equally same.

Reply:
Should be possible by marshalling/unmarshalling from/into a java object, but probably not worth the effort here.